doc/ovs-hw-offload.md

```diff
@@ -9,63 +9,137 @@ host net-device. The VF representor plays the same role as TAP devices
 in Para-Virtual (PV) setup. A packet sent through the VF representor on the host
 arrives to the VF, and a packet sent through the VF is received by its representor.
 
+OVS Hardware Offloading requires NIC to be configured in `switchdev` mode.
+
+The operator can automate the creation and configuration of OVS bridges when the "manageSoftwareBridges" function is activated.
+
 ## Supported Ethernet controllers
 
 The following manufacturers are known to work:
 
 - Mellanox ConnectX-5 and above
 
-## Instructions for Mellanox ConnectX-5
-
 ## Prerequisites
 
 - OpenVswitch installed
-- Network Manager installed
+
+### Activate "manageSoftwareBridges" featureGate
+
+```
+kubectl patch sriovoperatorconfigs.sriovnetwork.openshift.io -n network-operator default --patch '{ "spec": { "featureGates": { "manageSoftwareBridges": true  } } }' --type='merge'
+```
+
 
 ### Deploy SriovNetworkNodePolicy
 
 ```yaml
-apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetworkNodePolicy
 metadata:
-  name: ovs-hw-offload
-  namespace: sriov-network-operator
+  name: connectx6dx-switchdev
+  namespace: network-operator
 spec:
-  deviceType: netdevice
+  eSwitchMode: switchdev
+  mtu: 1500
   nicSelector:
-    deviceID: "1017"
-    rootDevices:
-    - 0000:02:00.0
-    - 0000:02:00.1
-    vendor: "15b3"
+    deviceID: 101d
+    vendor: 15b3
   nodeSelector:
-    feature.node.kubernetes.io/network-sriov.capable: "true"
-  numVfs: 8
-  priority: 10
-  resourceName: cx5_sriov_switchdev
-  isRdma: true
-  eSwitchMode: switchdev
-  linkType: eth
+    node-role.kubernetes.io/worker: ""
+  numVfs: 5
+  resourceName: switchdev
+  bridge:
+    ovs: {}
 ```
 
-### Create NetworkAttachmentDefinition CRD with OVS CNI config
+_Note: `spec.bridge.ovs: {}` will create OVS bridges with default configuration that is suitable for HW-offloading with OVS-kernel dataplane. 
```

Comment:
can you add that it will create a bridge for every device (PF) selected by the policy ?

i know its mentioned below as an observation of the node state but i think we need to put this info upfront.

Reply:
Yes, makes sense. I mentioned this explicitly in the section where I describe the policy.